test/blackbox-tests/test-cases/rewrite-tests/minlib.t

```diff
@@ -0,0 +1,62 @@
+Test the rewriting of a minimal library, and show that
+it does not conform to the locations where the artifacts
+are installed.
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.7)
+  > (name minlib)
+  > (package
+  >  (name minlib)
+  >  )
+  > EOF
+
+  $ mkdir lib
+
+  $ cat > lib/dune <<EOF
+  > (library
+  >  (name minlib)
+  >  (public_name minlib)
+  >  )
+  > (rule
+  >  (target x)
+  >  (action (system "dune_cmd rewrite-path $PWD/_build/default/lib/run.ml | grep -c '^/install_root/lib/minlib/run.ml$'; touch x;")))
```

Comment:
This test is missing an eof?

Reply:
Interesting. Apparently CRAM supplies a missing EOF. I've added the missing EOF and rebased.